### ISSUE TITLE
Can't drag and move the position of commands.

### ISSUE THREAD

**Issue by dingdonng22** [ChasKane/gay-toolbar#6]
Hi, before I could easily drag and relocate existing commands but today I couldn't do that. Kindly look into this.
https://github.com/user-attachments/assets/15e21426-5065-4555-b4d5-5465bdcd3b65

**Comment by ChasKane** [comment]
Hey, thanks for letting me know! I'm not able to replicate on android. Are you on android or iOS? Did you upgrade your phone's OS, your Obsidian version, or the plugin version? Are you on the latest plugin version?

Can you replicate on desktop? In the next version, I'll make sure errors are printed on screen. For now, if you can't replicate on desktop, you might be able to see any errors using the [mobile console plugin](https://github.com/KjellConnelly/obsidian-dev-tools).

**Comment by dingdonng22** [comment]
> Hey, thanks for letting me know! I'm not able to replicate on android. Are you on android or iOS? Did you upgrade your phone's OS, your Obsidian version, or the plugin version? Are you on the latest plugin version?

Yeah I'm on android. Be it the plugin or obsidian app, everything is in latest version. Hope you fix this in next update of your plugin.

**Comment by ChasKane** [comment]
I can't replicate this even after reinstalling from the latest version (I'm on android). I'll need more info to be able to debug this. There'll probably be console errors if you use the console plugin I linked. If that doesnt work, enable developer mode in your andoid settings, connect to a chromium browser like Chrome or Brave, and use [this built in browser page](brave://inspect/#devices) (replace `brave` with `chrome` (i think) for chrome) to inspect Obsidian running on your phone. there, you'll see the logs, any errors should show up in the console. send video of that if you can :)

Alternatively, ping me about scheduling a 1-1 debugging session LawrenceCKane@gmail.com

**Comment by ChasKane** [comment]
@dingdonng22 any updates?

**Comment by ChasKane** [comment]
@dingdonng22 lmk if you still see this issue.
